src/en/about/guide.md

```diff
@@ -15,6 +15,8 @@ Type | Description
 Active | Trending sort based on the score, and the most recent comment time.
 Hot | Trending sort based on the score, and the post creation time.
 New | Newest items.
+Most Comments | The posts with the most comments.
+New Comments | The posts that have new comments, IE a forum-style sort.
```

Comment:
"The posts that have most recently been commented on". Might be a bit clearer, but I'm not sure how to explain it.

Reply:
I'll say, "The posts with recent comments"

Reply:
"The post with *most* recent comments" sounds clearer to me.

Sorry for nitpicking so much haha

Reply:
Haha that's okay. updated